lib/caesar.rb

```diff
@@ -1,6 +1,42 @@
+require 'pry'
+
 class Caesar
-  attr_reader :key
 
-  def initialize
+  attr_reader :caesar
+              :eng_to_cipher
+
+ def initialize
+  end
+
+  def eng_to_cipher_lowercase(string, shift = 3)
+    alphabet = Array('a'..'z')
+    decrypt = Hash[alphabet.zip(alphabet.rotate(shift))]
+    string.chars.map do |letter|
+      decrypt.fetch(letter, " ")
+    end
+  end
+
+  def eng_to_cipher_all_case(string, shift = 3)
+    alphabet = Array('A'..'z')
```

Comment:
Think about how you could combine this with the lowercase method because they are all very similar